roles/keystone/meta/main.yml

```diff
@@ -14,15 +14,17 @@ dependencies:
     alternatives: "{{ keystone.alternatives }}"
     system_dependencies: "{{ keystone.source.system_dependencies }}"
     python_dependencies: "{{ keystone.source.python_dependencies }}"
+    python_post_deps: "{{ keystone.source.python_post_deps }}"
```

Comment:
sorry, original PR had it that way and i had commented to shorten it up. But is a quick change and give consistency.